integration/solana/simple_collectible.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: Apache-2.0
+
+// DISCLAIMER: This file is an example of how to mint and transfer NFTs on Solana. It is not production ready and has not been audited for security.
+// Use it at your own risk.
+
+import '../../solana-library/spl_token.sol';
+import '../../solana-library/system_instruction.sol';
+
+contract SimpleCollectible {
+    // Contract variables
+    uint32 private tokenCounter;
+    address private mint_authority;
+    address private payer;
```

Comment:
This is example is supposed to educate users how to create a NFT using Solidity, so everything needs commenting. Every variable and also explain solana concepts to Solidity users (maybe with links)